backends/projects/client.go

```diff
@@ -0,0 +1,148 @@
+package projects
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/ugent-library/biblio-backoffice/models"
+	"github.com/ugent-library/projects-service/api/v1"
+)
+
+type Config struct {
+	APIUrl string
+	APIKey string
+}
+
+type securitySource struct {
+	apiKey string
+}
+
+func (s *securitySource) ApiKey(ctx context.Context, operationName string) (api.ApiKey, error) {
+	return api.ApiKey{APIKey: s.apiKey}, nil
+}
+
+type Client struct {
+	config Config
+	client *api.Client
+}
+
+func New(c Config) (*Client, error) {
+	client, err := api.NewClient(c.APIUrl, &securitySource{c.APIKey})
+	if err != nil {
+		return nil, err
+	}
+
+	return &Client{
+		config: c,
+		client: client,
+	}, nil
+}
+
+func (c *Client) GetProject(id string) (*models.Project, error) {
+	ctx := context.Background()
```

Comment:
better to use context.TODO() if it's a placeholder

Reply:
Done.